File: JAVA Solutions/CustomerWealth1672.java
class Solution1 {
    public int maximumWealth(int[][] accounts) {
        int min=Integer.MIN_VALUE;
        for(int[] r:accounts){
            int rowSum=0;
            for(int c:r){
                rowSum+=c;    
            }
            if(rowSum>min){
                min=rowSum;
            }
        }
        return min;
    }
}
/* second method using normal for loop */

class Solution2 {
    public int maximumWealth(int[][] accounts) {
        int min=Integer.MIN_VALUE;
        for(int[] r:accounts){
            int rowSum=0;
            for(int c:r){
                rowSum+=c;    
            }
            if(rowSum>min){
                min=rowSum;
            }
        }
        return min;
    }
}

File: C++ Solutions/CustomerWealth1672.cpp
class Solution {
public:
    int maximumWealth(vector<vector<int>>& accounts) {
        int min=INT_MIN;
        for(int r=0; r<accounts.size(); r++){
            int rowSum=0;
            for(int c=0;c<accounts[r].size();c++){
                rowSum+=accounts[r][c];    
            }
            if(rowSum>min){
                min=rowSum;
            }
        }
        return min;   
    }
};